### ISSUE TITLE
Strange results for licenses issue

### ISSUE THREAD

**Issue by GantMan** [davglass/license-checker#137]
hunspell-spellchecker has a very clear and simple license... yet this is what came out:
```json
  "hunspell-spellchecker@1.0.2": {
    "licenses": "Custom: https://travis-ci.org/GitbookIO/hunspell-spellchecker.png",
    "repository": "https://github.com/GitbookIO/hunspell-spellchecker",
    "publisher": "GitBook",
    "email": "contact@gitbook.com",
    "path": "/Users/gantman/Documents/Projects/playground/solidarity/node_modules/hunspell-spellchecker",
    "licenseFile": "/Users/gantman/Documents/Projects/playground/solidarity/node_modules/hunspell-spellchecker/README.md"
  },
```

same goes for `log-driver@1.2.5`
```json
  "log-driver@1.2.5": {
    "licenses": "Custom: http://linuxcommand.org/man",
    "repository": "https://github.com/cainus/logdriver",
    "path": "/Users/gantman/Documents/Projects/playground/solidarity/node_modules/log-driver",
    "licenseFile": "/Users/gantman/Documents/Projects/playground/solidarity/node_modules/log-driver/README.md"
  },
```

They clearly define their license, but the wrong thing is detected.

**Comment by davglass** [comment]
`log-driver@1.2.5` does not have a license in the tarball they deploy to npm:

```
log-driver $ ls
Makefile     README.md    index.js     package.json
```

Neither does `hunspell-spellchecker@1.0.2`:

```
hunspell-spellchecker $ ls
README.md    bin          gulpfile.js  lib          package.json test
```

`hunspell-spellchecker` does have a key in the `package.json` (`"license": "Apache 2",`) I need to look and see why this wasn't picked up as a valid license tho.

**Comment by GantMan** [comment]
how do you access the tarball?  Also, how cool would it be to check the github for missing licenses?
as a feature.

**Comment by davglass** [comment]
You can also just hit the registry:

http://registry.npmjs.org/license-checker/latest

Under the `dist.tarball` key.

That actually defeats the point, if the code is shipped without a license, it's technically unlicensed. The license must ship with the code or it's not covered. I have intentionally never added this feature, by design this only scans what's locally installed.

**Comment by honzajavorek** [comment]
I think this should be closed. The packages missing licenses in their npm distributions need to fix their npm packages contents, there's nothing the license checker should do about this. They effectively distribute unlicensed code.
